api/src/pcapi/core/offerers/repository.py

```diff
@@ -774,3 +776,66 @@ def get_number_of_pending_collective_offers_for_offerer(offerer_id: int) -> int:
         .limit(MAX_OFFERS_PER_OFFERER_FOR_COUNT)
         .count()
     )
+
+
+def _get_revenues_per_year(
```

Comment:
Détail : la fonction n'est pas privée au module puisque appelée depuis les blueprints
```suggestion
def get_revenues_per_year(
```